docs/change-log.md

```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update October 20th, 2023
+
+- **New:** The Humanitec Terraform Driver is now in General Availability! This major release allows more seamless integrations and management of infrastructure as code (IaC) with Terraform. Check out the [feature announcement](https://humanitec.com/blog/the-humanitec-terraform-driver-enters-general-availability) for more details.
```

Comment:
```suggestion
- **New:** The Humanitec Terraform driver is now at general availability (GA). Following a successful experimental beta phase, we have addressed implementation gaps, resulting in a more robust and reliable user experience. Check out the [feature announcement](https://humanitec.com/blog/the-humanitec-terraform-driver-enters-general-availability) for more details.
```

Reply:
Suggest: "is now generally available" instead of "is now at general availability"